script/migrations/21-multiple_upgrade.sol

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.18;
+
+import "../../src/wallet/KintoWalletFactory.sol";
+import "../../src/wallet/KintoWallet.sol";
+import "../../src/apps/KintoAppRegistry.sol";
+import "../../src/paymasters/SponsorPaymaster.sol";
+import "../../src/viewers/KYCViewer.sol";
+import "../../src/KintoID.sol";
+
+import "../../test/helpers/Create2Helper.sol";
+import "../../test/helpers/ArtifactsReader.sol";
+import "../../test/helpers/UUPSProxy.sol";
+import "../../test/helpers/UserOp.sol";
+
+import "forge-std/Script.sol";
+import "forge-std/console.sol";
+
+contract KintoMigration21DeployScript is Create2Helper, ArtifactsReader, UserOp {
+    using ECDSAUpgradeable for bytes32;
+
+    KintoWalletFactory _walletFactory;
+    uint256 deployerPrivateKey;
+
+    // NOTE: this migration must be run from the ledger admin
+    function run() public {
+        console.log("RUNNING ON CHAIN WITH ID", vm.toString(block.chainid));
+
+        deployerPrivateKey = vm.envUint("PRIVATE_KEY");
+
+        // execute this script with the with the ledger
+        console.log("Executing from address", msg.sender);
+
+        // set wallet factory
+        _walletFactory = KintoWalletFactory(payable(_getChainDeployment("KintoWalletFactory")));
+
+        // deploy contracts
+        SponsorPaymasterV4 _paymasterImpl = SponsorPaymasterV4(upgradePaymaster());
+        KintoAppRegistryV3 _registryImpl = KintoAppRegistryV3(upgradeRegistry());
+        KintoWalletV4 _walletImpl = KintoWalletV4(payable(upgradeWallet()));
+        KintoWalletFactoryV7 _factoryImpl = KintoWalletFactoryV7(upgradeFactory());
+        KYCViewerV2 _kycViewerImpl = KYCViewerV2(upgradeKYCViewer());
```

Comment:
need to redeploy kyc viewer completely. The previous one is not usable because we didnt call disableInitializers